src/fragments/dropdowns-filtering-notes.mdx

```diff
@@ -6,5 +6,6 @@
 
 - The dropdowns package does not have filtering logic. The simplest way to include filtering is to control
   the `inputValue` and `onInputValueChange` props, and conditionally render Item components as necessary.
+- Use the `onInputValueChange` prop instead of `onStateChange`, for more accurate IME support
```

Comment:
Does downshift fire `onInputValueChange` for all the [stateChangeTypes](https://github.com/downshift-js/downshift#statechangetypes) listed? I haven't tested, but I'm thinking it doesn't, which might be why they document with "...use instead or in combination with `onStateChange`..." verbiage for [onInputValueChange](https://github.com/downshift-js/downshift#oninputvaluechange).

For example, it could be a misdirect if a consumer believes they should use `onInputValueChange` in order to capture a `Downshift.stateChangeTypes.blurButton` event.

Reply:
Perhaps an easy fix would be...

"Use the `onInputValueChange` prop instead of or in combination with `onStateChange`, ..."

Reply:
Ah, I see what you mean. Let me update it.